code/src/components/Thoughts form/ThoughtsForm.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './style.css'
+
+const ThoughtsForm = ({ newThought, handleNewThoughtChange, onFormSubmit }) => {
+  return (
+    <div >
+      <form className='styleform' onSubmit={onFormSubmit}>
+        <h1>Welcome to happy thoughts app! Type new thought below.</h1>
+        <textarea 
+        className='textArea'
+        value={newThought}
+        onChange={handleNewThoughtChange}
+        placeholder='Enter yout thought here...'
+        maxLength="140" />
+        <div className="counter">{0 + newThought.length} / {140 - newThought.length} </div>
```

Comment:
Creative way of making the thought count.  Nicely thought of how you did the counter. the option in react  also called require can be used for these =)